src/libraries/System.Linq.Expressions/src/System/Dynamic/Utils/TypeUtils.cs

```diff
@@ -623,7 +623,8 @@ public static bool HasBuiltInEqualityOperator(Type left, Type right)
             || IsImplicitBoxingConversion(source, destination)
             || IsImplicitNullableConversion(source, destination);
 
-        [RequiresUnreferencedCode(Expression.ExpressionRequiresUnreferencedCode)]
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2075:UnrecognizedReflectionPattern",
+            Justification = "The trimmer doesn't remove operators when System.Linq.Expressions is used. See https://github.com/mono/linker/issues/1821.")]
```

Comment:
Nit: that issue is still open... should this link to https://github.com/mono/linker/pull/2125 instead?

Reply:
@sbomer @vitek-karas - any reason that issue is still open? Are we keeping it open for the “long term plan”? Although, the long term plan is more runtime and Roslyn work, and not really linker work.

I can change the link to the PR if that makes more sense.

Reply:
I would change it to the PR - I'd like to keep the linker issue opened to track the potential future work on roslyn/runtime.